test/object-store/realm.cpp

```diff
@@ -2309,7 +2310,9 @@ TEST_CASE("Realm::delete_files()") {
         REQUIRE(util::File::exists(path + ".lock"));
         REQUIRE(util::File::exists(path));
         REQUIRE(util::File::exists(path + ".management"));
+#ifndef _WIN32
         REQUIRE(util::File::exists(path + ".note"));
+#endif
```

Comment:
This is actually a correct exclusion - Object Store's `ExternalCommitHelper` implementation on Windows doesn't use named pipes (and named pipes on Windows aren't filesystem objects anyway) so `.note` files never exist on Windows.